app/lti_controller.rb

```diff
@@ -95,13 +96,16 @@ class LtiController < Sinatra::Base
       consumer_secret: shared_secret,
       callback: 'about:blank'
     }
+
+    # this key is from sinatra, which messes up oauth signature validation
+    params.delete(:captures)
```

Comment:
Did this come from somewhere else? Seems like kind of a random change for this pull request

Reply:
it's because of the sinatra 1 -> 2 upgrade.  sinatra 2 adds this key to params in every request representing the named captures from the route (which are empty, in this case).